imminence/redis.yml

```diff
@@ -0,0 +1,4 @@
+# this file gets overriden on deploy
+host: 127.0.0.1
+port: 6379
+namespace: imminence-development
```

Comment:
It's a dash in imminence's redis.yml - https://github.com/alphagov/imminence/blob/master/config/redis.yml#L5 - maybe it should be an underscore there for consistency, but it's not :(